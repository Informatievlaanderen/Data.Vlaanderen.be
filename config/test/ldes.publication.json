[
   {
      "urlref": "/doc/applicatieprofiel/ldes/ontwerpstandaard/2022-05-04",
      "repository": "https://github.com/Informatievlaanderen/OSLOthema-ldes",
      "branchtag": "master",
      "name": "OSLO-LDES-ap",
      "filename": "config/OSLO-LDES-ap.json",
      "navigation": {},
      "dummy": "044"
   },
   {
      "urlref": "/doc/vocabularium/ldes/ontwerpstandaard/2022-05-04",
      "repository": "https://github.com/Informatievlaanderen/OSLOthema-ldes",
      "branchtag": "master",
      "name": "OSLO-LDES-voc",
      "filename": "config/OSLO-LDES-voc.json",
      "navigation": {},
      "dummy": "044"
   },
   {
      "urlref": "/doc/applicatieprofiel/ldes",
      "seealso": "/doc/applicatieprofiel/ldes/ontwerpstandaard/2022-05-04",
      "type": "ap",
      "dummy": "044"
   },
   {
      "urlref": "/ns/ldes",
      "seealso": "/doc/vocabularium/ldes/ontwerpstandaard/2022-05-04",
      "type": "voc",
      "dummy": "044"
   }
]
