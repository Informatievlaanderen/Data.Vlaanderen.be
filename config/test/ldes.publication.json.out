[
   {
      "urlref": "/doc/applicatieprofiel/ldes/ontwerpstandaard/2022-05-17",
      "repository": "https://github.com/Informatievlaanderen/OSLOthema-ldes",
      "branchtag": "master",
      "name": "OSLO-LDES-ap",
      "filename": "config/OSLO-LDES-ap.json",
      "navigation": {},
      "dummy": "058",
      "disabled": false
   },
   {
      "urlref": "/doc/vocabularium/ldes/ontwerpstandaard/2022-05-17",
      "repository": "https://github.com/Informatievlaanderen/OSLOthema-ldes",
      "branchtag": "master",
      "name": "ldes",
      "filename": "config/OSLO-LDES-voc.json",
      "navigation": {},
      "dummy": "058",
      "disabled": false
   },
   {
      "urlref": "/doc/applicatieprofiel/ldes",
      "seealso": "/doc/applicatieprofiel/ldes/ontwerpstandaard/2022-05-16",
      "type": "ap",
      "dummy": "057",
      "disabled": false
   },
   {
      "urlref": "/ns/ldes",
      "seealso": "/doc/vocabularium/ldes/ontwerpstandaard/2022-05-16",
      "type": "voc",
      "dummy": "057",
      "disabled": false
   }
]
